.github/create_opm_index.sh

```diff
@@ -9,8 +9,8 @@ echo "${INDEX_IMAGE_TAG}"
 echo "${BUNDLE_IMAGE}"
 echo "${STORAGE_BUNDLE_IMAGE}"
 
-echo "opm index add --bundles "${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA}","${REGISTRY}/${STORAGE_BUNDLE_IMAGE}:${GITHUB_SHA}" --tag "${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA}" -u podman --pull-tool podman"
-opm index add --bundles "${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA}","${REGISTRY}/${STORAGE_BUNDLE_IMAGE}:${GITHUB_SHA}" --tag "${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA}" -u podman --pull-tool podman
+echo "opm index add --bundles "${REGISTRY}/${BUNDLE_IMAGE}:${GITHUB_SHA}","${REGISTRY}/${STORAGE_BUNDLE_IMAGE}:${GITHUB_SHA}","quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle@sha256:9ff91ad3c9ef1797b232fce2f9adf6ede5c3421163bff5b8a2a462c6a2b3a68b" --tag "${REGISTRY}/${INDEX_IMAGE}:${GITHUB_SHA}" -u podman --pull-tool podman"
```

Comment:
+1 in general, and given you put here the rabbit bundle w/ its sha I suspect we don't rebuild it often. Is this a short term fix (e.g., until we create the `messaging` bundle for osp) or are you leaving `cluster-operator` by sha on purpose?

Anyway approving as it's a short term fix that unblock people, we can follow up on that front

Reply:
this is a short term fix. But I do also want to discuss "what if we just let all the bundles be for individual operators"?

Reply:
ack makes sense, let's merge this first and I'm +1 to follow up and put all the options on the table.